google/api_versions.go

```diff
@@ -0,0 +1,99 @@
+package google
```

Comment:
Are license headers something that we include in any of these files? Looking over `provider.go`, `config.go`, and a few resources, I can't see one included elsewhere.